src/content/docs/alerts-applied-intelligence/applied-intelligence/anomaly-detection/anomaly-detection-applied-intelligence.mdx

```diff
@@ -162,78 +164,63 @@ These events are also available for querying, creating custom dashboards, and al
-
-Applications will not always generate anomalies, so it can be normal to not receive any detections.
-
-### Requirements [#requirements]
-
-To use New Relic's anomaly detection, ensure you have:
+<CollapserGroup>
+  <Collapser
+    className="freq-link"
+    id="jira"
+    title="Requirements"
+  >
 
-* An APM agent installed for at least one application.
-* To receive notifications in Slack, you'll need to ask your IT administrator to install the New Relic application in [your Slack workspace](https://slack.com/help/articles/202035138-Add-an-app-to-your-workspace).
+  To use New Relic's anomaly detection, ensure you have:
 
-For more on data limits, see [Data limits](#data-limits).
+  * An APM agent installed for at least one application.
+  * To receive notifications in Slack, you'll need to ask your IT administrator to install the New Relic application in [your Slack workspace](https://slack.com/help/articles/202035138-Add-an-app-to-your-workspace).
 
-### Set up notifications for anomaly detection [#set-up]
+  For more on data limits, see [Data limits](#data-limits).
+  </Collapser>
 
-Automatic anomaly detection is enabled automatically, at no additional cost. To receive notifications or to have a configuration (group of apps) that you can add as a source for incident intelligence, you will need to create a anomaly detection configuration. 
+    <Collapser
+    className="freq-link"
+    id="jira"
+    title="Notifications"
+  >
+  Anomaly detection is enabled automatically, at no additional cost. To receive notifications or to have a configuration (group of apps) that you can add as a source for incident intelligence, you will need to create a anomaly detection configuration. 
 
-You can create a configuration in the anomaly detection UI:
+  You can create a configuration in the anomaly detection UI:
 
-1. From **[one.newrelic.com](https://one.newrelic.com/all-capabilities)**, click **Alerts & AI**.
-2. Under **Anomaly detection**, click **Settings**.
-3. Click **Add a configuration**.
-4. Input the following information into the form:
+  1. From **[one.newrelic.com](https://one.newrelic.com/all-capabilities)**, click **Alerts & AI**.
+  2. Under **Anomaly detection**, click **Settings**.
+  3. Click **Add a configuration**.
+  4. Input the following information into the form:
 
    * Choose a name for your configuration that helps you easily identify it from others in your account.
    * Select an account.
    * Select up to 1,000 applications. Note that certain applications with low throughput might not be good candidates for anomaly detection, as they can be more sensitive to smaller amounts of data fluctuation.
-5. Optional: select the golden signals you'd like to monitor for anomalies.
-6. Optional: [connect to incident intelligence](#add-source).
+  5. Optional: select the golden signals you'd like to monitor for anomalies.
```

Comment:
Do we mention what the golden signals are before this? I don't remember seeing it, just wanted to bring it up to prevent any confusion. If we don't mention it maybe we could just throw a link in here.